code/go/internal/validator/folder_item_spec.go

```diff
@@ -79,8 +79,6 @@ func (s *folderItemSpec) validate(fs fs.FS, folderSpecPath string, itemPath stri
 	if s.Ref != "" {
 		schemaPath := filepath.Join(filepath.Dir(folderSpecPath), s.Ref)
 		schemaLoader = yamlschema.NewReferenceLoaderFileSystem("file:///"+schemaPath, fs)
-	} else if s.Content != nil {
```

Comment:
I'm trying to remind why did we need this attribute and IIRC it's to embed the spec instead of referencing. For example:

```
  - description: The package's CHANGELOG file
    type: file
    contentMediaType: "application/x-yaml"
    name: "changelog.yml"
    required: true
    $ref: "./changelog.spec.yml"
```

could be replaced with:

```
  - description: The package's CHANGELOG file
    type: file
    contentMediaType: "application/x-yaml"
    name: "changelog.yml"
    required: true
    content:
        <content of changelog.spec.yml, but embedded>
```

Thanks for checking if we're using this property in spec and if we don't, I guess it's safe to get rid of it.

Reply:
Umm, well, we can always revert this change if we find a situation where we need embedded specs :slightly_smiling_face:

Reply:
I might be wrong, but it was intended for really short spec files. I guess it's safe to remove it :)